### DISCUSSION TITLE
Add prettier-plugin-solidity as default

### DISCUSSION THREAD

**Discussion by Robsonsjre** [paulrberg/foundry-template#33] [Q&A] [answered]
I saw that you are using Solhint in the default template, any reason why not add prettier-plugin-solidity pointing to the Solidity contracts too 🤔

**Comment by PaulRBerg** [comment, marked as answer]
Hi @Robsonsjre, this is because I am using [`forge fmt`](https://book.getfoundry.sh/reference/config/formatter?highlight=formatte#formatter) now. See the config [here](https://github.com/PaulRBerg/foundry-template/blob/df980088181d556a01a2edf01546cd9e112b2447/foundry.toml#L34-L42).
